lib/matplotlib/text.py

```diff
@@ -2159,11 +2158,17 @@ def _update_position_xytext(self, renderer, xy_pixel):
                         " use 'headlength' to set the head length in points.")
                 headlength = d.pop('headlength', 12)
 
-                to_style = self.figure.dpi / (72 * ms)
-
-                stylekw = dict(head_length=headlength * to_style,
-                               head_width=headwidth * to_style,
-                               tail_width=width * to_style)
+                # Old way:
```

Comment:
please remove this comment block

If anyone has to do forensics on this part of the code recursive git-blame is their friend.   I think going forward the discussion of old and new way will just lead to confusion